zeppelin-interpreter/src/main/java/org/apache/zeppelin/scheduler/Job.java

```diff
@@ -86,6 +86,7 @@ public boolean isPending() {
   private transient Throwable exception;
   private transient JobListener listener;
   private long progressUpdateIntervalMs;
+  protected Map<String, Object> infos;
```

Comment:
If there're no special usage of this field, let's remove it. Fields in Job.java, Paragraph.java and Note.java will be persisted into note.json.